model/execution/DeliveryExecutionManagerService.php

```diff
@@ -285,7 +299,15 @@ public function adjustTimers(array $deliveryExecutions, $seconds)
 
                 $data = $deliveryMonitoringService->getData($deliveryExecution);
                 $data->updateData([DeliveryMonitoringService::REMAINING_TIME]);
+
                 $deliveryMonitoringService->save($data);
+
+                // log adjusted time of the de
+                $adjustmentLog = [
+                    'reason' => $reason,
+                    'increment' => $seconds,
+                ];
+                $this->getDeliveryLogService()->log($deliveryExecution->getIdentifier(), 'ADJUSTED_TIME', $adjustmentLog);
```

Comment:
I understand that it's implemented in the same way it's done for pause/terminate requests. But in general, I think it should not happen here. I think it would be better to trigger event time adjusted and in specific listener handle this event - store log record. In this way, our code will follow the single responsibility principle and we will be able easier to handle these events asynchronously if we decide in the future.
Not critical but could be a bit better solution I think.

Reply:
Agree, but now I've added the log, if I do as you suggested, then I need to refactor all this class and create a lot of php unit tests, not sure that this is required by the task

Reply:
I support the suggestion by @SashaTAO, I think we were discussing this approach during planning, when discussing ticket `TCA-402`.

Reply:
done